debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -232,10 +249,18 @@ protected void execute() {
                         if (filters.tableFilter().test(id)) {
                             tableIds.add(id);
                             tableIdsByDbName.computeIfAbsent(dbName, k -> new ArrayList<>()).add(id);
+                        } else if (logger.isTraceEnabled()) {
+                            logger.trace("'{}' is filtered out, discarding", id);
                         }
                     }
                 });
             }
+            if (logger.isDebugEnabled()) {
+                String tablesByDbName = tableIdsByDbName.values().stream()
+                                                        .map(Object::toString)
+                                                        .collect(Collectors.joining(System.lineSeparator()));
+                logger.debug("Step 5: using tables: {}", tablesByDbName);
+            }
```

Comment:
I think it would be better to not prefix this with "Step 5" (since there already was one of those log statements), and instead add the debug log output after line 251 when it is added to `tableIds`. It's okay that each table included will result in a separate log statement, and it's much easier to see them when there are more than a few tables.